### DISCUSSION TITLE
Search icon on the taskbar hook (Windows 10)

### DISCUSSION THREAD

**Discussion by bywkgexmkiltxh** [valinet/ExplorerPatcher#3216] [Ideas]
**Start** button style *: Windows 10 (default) / Windows 11
Windows 10 **search** icon got changed with KB5025297, can you implement the same feature as the start button style above, for search icon? So it'll be before April 25, 2023 patch
![search](https://github.com/valinet/ExplorerPatcher/assets/127624994/23e46224-498b-4ba1-b2a5-fb05cef8fc5d)

**Comment by pyrates999** [comment]
I think this is beyond what EP is for, sorry.

**Reply by bywkgexmkiltxh** [reply]
But explorerpatcher can replace the start button, why can't it do the same for the search button
